ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -47,6 +47,10 @@ SettingsContentBase {
         id: stackContainer
 
         width: root.contentWidth
+        height: stackContainer.currentIndex === root.mainViewIndex ? main.height:
```

Comment:
@Khushboo-dev-cpp setting height to `childrenRect.height` instead of all these, doesn't do the thing?